client/components/web-preview/style.scss

```diff
@@ -297,12 +299,30 @@ a.web-preview__external.button {
 	left: initial;
 }
 
+.web-preview__loading-message-wrapper {
+	display: flex;
+	justify-content: center;
+	height: 100%;
+	flex-direction: column;
+	position: relative;
+	z-index: z-index( 'root', '.web-preview__loading-message-wrapper' );
+}
+
+.web-preview__loading-message-wrapper .spinner__image {
+	margin: 0 auto 16px;
+}
+
 .web-preview__loading-message {
-	color: $gray;
-	font-size: 18px;
+	color: $gray-text-min;
 	font-weight: 300;
 	text-align: center;
 	display: block;
-	margin: 48px 0;
-	height: 100%;
+	font-size: 16px;
+}
+
+.web-preview__loading-message strong {
```

Comment:
In general, you are totally right. In this case that would, however, mean that components using WebPreview that want to declare a loading message would need to use WebPreview internal classname in order to get a proper styling. Thus we opted to provide the option to use strong and we'll take care of styling internally.

An alternative to this approach (more correct but more constraining) would be to split the prop into two - something like `loadingMessageHeading` & `loadingMessageDescription` and then internally wrap them in elements with correct classnames.